minitests/litex/uart_ddr/arty/src.yosys/missing_bit_report.py

```diff
@@ -0,0 +1,81 @@
+""" Generates a missing feature/bit report for LiteX design.
```

Comment:
I think this one should be implemented as more generic tool. Leaving it here will shortly end up in notworking tool nobody uses.

Reply:
Hmm, I believe this is not needed, it is actually a leftover from my local branch.

This script should be relative to only HCLK_IOI and IOI3 tiles unknown features, which are solved by now.